service/src/test/java/tds/assessment/repositories/impl/AdminSubjectQueryRepositoriesIntegrationTests.java

```diff
@@ -1,41 +0,0 @@
-package tds.assessment.repositories.impl;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-import org.springframework.transaction.annotation.Transactional;
-
-import java.util.Optional;
-
-import tds.assessment.SetOfAdminSubject;
-import tds.assessment.repositories.AdminSubjectQueryRepository;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-@RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest
-@Transactional
-public class AdminSubjectQueryRepositoriesIntegrationTests {
```

Comment:
True, I am hoping that the service integration tests would make sure to hit those conditionals.